server/routerlicious/packages/routerlicious-base/src/riddler/tenantManager.ts

```diff
@@ -39,6 +39,10 @@ export interface ITenantDocument {
 
     // Whether the tenant is disabled
     disabled: boolean;
+
+    // Timestamp of when this tenant will be hard deleted.
+    // Only applicable if the tenant is disabled.
+    deletionTime?: string;
```

Comment:
Until I found the comment here, I was confused because I thought "deletionTime" meant time that it _was_ deleted not time that it _will be_ deleted. Possibly just me not having enough context. My suggestion though would be to rename to `scheduledDeletionTime`